packages/js/src/plugins/auctionHouseModule/operations/withdrawFromBuyerAccount.ts

```diff
@@ -70,19 +73,30 @@ export type WithdrawInput = {
 
   /**
    * The buyer who withdraws funds.
-   * This expects a Signer.
+   *
+   * There must be one and only one signer; Authority or Seller must sign.
+   *
+   * @defaultValue `metaplex.identity()`
+   */
+  buyer?: PublicKey | Signer;
+
+  /**
+   * The Signer paying for the creation of all accounts
+   * required to deposit to the buyer's account.
+   * This account will also pay for the transaction fee.
    *
    * @defaultValue `metaplex.identity()`
    */
-  buyer?: Signer;
+  payer?: Signer;
 
   /**
    * The Authority key.
    * It is required when the buyer is not a signer.
+   * There must be one and only one signer; Authority or Buyer must sign.
    *
    * @defaultValue Defaults to not being used.
    */
-  authority?: Signer;
+  authority?: PublicKey | Signer;
```

Comment:
Again, this does not need to be a `PublicKey`. We still potentially require this as a Signer since either the authority or the buyer needs to sign the transaction but we already know the authority as a PublicKey from the `auctionHouse.authorityAddress` parameter.

It should be:

```ts
authority?: Signer;
```